spanner/spannertest/db.go

```diff
@@ -68,6 +68,7 @@ type table struct {
 type colInfo struct {
 	Name     spansql.ID
 	Type     spansql.Type
+	NotNull  bool
```

Comment:
given this structure is generally populated in a few different places for different purposes, and `NotNull` is only going to be set for table columns, please add a line column to that effect.

```go
  NotNull bool  // only set for table columns
```

Reply:
also, update the code near line 222 to return this.

Reply:
I see. And, thanks for letting me know the missing place to update.

https://github.com/googleapis/google-cloud-go/pull/3491/commits/931c0baa70312dd195320deb77114ae3ce7532db